build/debian/control.precise

```diff
@@ -30,7 +30,7 @@ Build-Depends: debhelper (>= 4.0.0),
                libusb-1.0-0-dev, 
                libchromaprint-dev, 
                librubberband-dev, 
-               libsqlite3-dev
+               libsqlite3-dev,
 Standards-Version: 3.7.3
```

Comment:
Precise has no Hidapi, thats why I had to do an extra cycle in this PR.
Precise builds fall back to the hidapi version in the Mixxx source tree.
The only issue is the stray comma ... fixed.